### PR TITLE
Modified Node agent tags to be hierarchical.

### PR DESCRIPTION
**Pull request by nnDarshan**
This patch modifies node_agent tags to be hierarchical.

### CODE REVIEW

**Comment by nnDarshan** [comment]
@shtripat @r0h4n @brainfunked @nthomas-redhat Please review. Node agent tags have been changed as per the discussion yesterday

File: tendrl/node_agent/manager/pull_service_status.py [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 TENDRL_SERVICE_TAGS = {
-    "tendrl-node-agent": "TENDRL",
-    "etcd": "TENDRL_SERVER",
-    "tendrl-apid": "TENDRL_SERVER",
-    "tendrl-gluster-integration": "TENDRL_GLUSTER",
-    "tendrl-ceph-integration": "TENDRL_CEPH",
-    "glusterd": "GLUSTER",
-    "ceph-mon": "CEPH_MON",
-    "ceph-osd": "CEPH_OSD"
+    "tendrl-node-agent": "TENDRL/NODE",
```
Comment (shtripat) [inline tendrl/node_agent/manager/pull_service_status.py]: Should we really keep the names uppercase? I feel these would becomes keys in etcd as well. So we might make them lowercase for better typing.. suggestions?

Comment (nnDarshan) [inline tendrl/node_agent/manager/pull_service_status.py]: these are values in etcd. So this should fine I feel.

Comment (shtripat) [inline tendrl/node_agent/manager/pull_service_status.py]: ok

Comment (brainfunked) [inline tendrl/node_agent/manager/pull_service_status.py]: Let's downcase this. These would probably end up becoming keys when indexed.

Comment (nnDarshan) [inline tendrl/node_agent/manager/pull_service_status.py]: Okay, will change to lowercase

**Pull request closed without merging**